src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/EditIssueStep.java

```diff
@@ -93,7 +95,6 @@ protected Execution(final EditIssueStep step, final StepContext context)
 
         if (issue == null) {
           errorMessage = "issue is null.";
-          return buildErrorResponse(new RuntimeException(errorMessage));
```

Comment:
Why did we remove this line?

Reply:
This return statement is redundant, as the error message is built and returned in lines 100-104.